securecookies/middleware.py

```diff
@@ -138,27 +168,11 @@ async def dispatch(
         # propagate the modified request
         response: Response = await call_next(request)
 
-        # for every cookie in the response
-        for cookie in response.headers.getlist("set-cookie"):
```

Comment:
re greater comment, just adding
```python
cookies = response.headers.getlist("set-cookie")
del response.headers["set-cookie"]

for cookie in cookies:
```
?

Reply:
I think you might be right. I'll try this without changing the test and check it works. I agree it'd be much better to have a smaller diff.

Reply:
@RealOrangeOne thought about it a little more. you'll need to tweak my suggestion a bit in order to preserve nonincluded and excluded cookies -- not sure what will look cleaner (your approach vs `set_cookie`) taking that into consideration

Reply:
I think the main issue would be `set_cookie` needing to account both for overrides and non-overrides. I suspect my approach is simpler. I'll make your other suggestions and we can see how the diff looks.